config.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "pluginAlias": "Eufy",
+  "pluginType": "platform",
+  "singular": true,
+  "schema": {
+    "type": "object",
+    "properties": {
+      "platform": {
+        "title": "Platform",
+        "type": "string",
+        "default": "eufy",
+        "required": true
+      },
+      "email": {
+        "title": "Email",
+        "type": "string",
+        "required": true,
+        "description": "Your Eufy Home email."
+      },
+      "password": {
+        "title": "Password",
+        "type": "string",
+        "required": true,
+        "description": "Your Eufy Home password."
```

Comment:
I would add the following, to ensure the password is at least secured a little bit. (It looks like `password` does not exist as a type.) But setting it as hidden (which I assume adds an option to show the field, but doesn't show it by default) is what the [HOOBS sample configuration schema](https://support.hoobs.org/docs/5e764eb6e87d1e02b6c19d4e) does, too.

```
"options": {
    "hidden": true
}
```